.travis.yml

```diff
@@ -89,6 +89,7 @@ env:
       MATRIX_EVAL="CC=gcc-9 CXX=g++-9"
     - BUILD_MODE=cmake
       NRN_ENABLE_PYTHON_DYNAMIC=ON
+      PIP2="$(which pip2)"
       MATRIX_EVAL="CC=gcc-8 CXX=g++-8"
```

Comment:
Does this build configuration compiles neuron with python2? I was expecting this to be python3 if python is not specified.

Reply:
I've noticed that as well (though I guess it is a separate issue). I haven't been able to figure out how to get python3 to take precedence over python2 with cmake (although that happens with dynamic loading and putting python3 after python2 in the
-DNRN_PYTHON_DYNAMIC list)

Reply:
It seems this pull request is ready to merge after I update the branch. Yes? @ramcdougal @pramodk @adamjhn

Reply:
Ok, I see now what's happening - python3 is detected first but python2 is also added with dynamic python support! So we need numpy for python2 as well.


```
-- Configured NEURON 8.0
487-- 
488-- Some things you can do now:
489-- --------------+--------------------------------------------------------------
490-- Command       |   Description
491-- --------------+--------------------------------------------------------------
492-- make install  | Will install NEURON to: /Users/travis/install
493--               | Change the install location of NEURON using:
494--               |     cmake <src_path> -DCMAKE_INSTALL_PREFIX=<install_path>
495-- make docs     | Build full docs. Calls targets: doxygen, prepare_docs, sphinx
496-- make uninstall| Removes files installed by make install (todo)
497-- --------------+--------------------------------------------------------------
498--  Build option | Status
499-- --------------+--------------------------------------------------------------
500-- C COMPILER    | /Applications/Xcode-11.3.1.app/Contents/Developer/usr/bin/gcc
501-- CXX COMPILER  | /Applications/Xcode-11.3.1.app/Contents/Developer/usr/bin/g++
502-- BUILD_TYPE    | RelWithDebInfo (allowed: Custom;Debug;Release;RelWithDebInfo;Fast)
503-- COMPILE FLAGS | -g  -O2
504-- Shared        | ON
505-- Binary special| ON
506-- Legacy FR     | ON
507-- MPI           | ON
508--   INC         | /usr/local/Cellar/open-mpi/4.0.3/include
509--   LIB         | /usr/local/Cellar/open-mpi/4.0.3/lib/libmpi.dylib
510--   DYNAMIC     | OFF
511-- Python        | ON
512--   EXE         | /usr/local/bin/python3
513--   INC         | /usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/include/python3.7m
514--   LIB         | /usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/libpython3.7m.dylib
515--   MODULE      | ON
516--   DYNAMIC     | ON
517--     EXE       | /usr/local/bin/python2
518--     INC       | /usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/include/python2.7
519--     LIB       | /usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/lib/libpython2.7.dylib
520--     INSTALL C | /usr/local/bin/python2 setup.py install --prefix= --home=/Users/travis/install
521--     EXE       | /usr/local/bin/python3
522--     INC       | /usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/include/python3.7m
523--     LIB       | /usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/libpython3.7m.dylib
524--     INSTALL C | /usr/local/bin/python3 setup.py install --prefix= --home=/Users/travis/install
525-- Readline      | /Applications/Xcode-11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/usr/lib/libreadline.tbd
526-- Curses        | /usr/lib/libcurses.dylib;/usr/lib/libform.dylib
527-- RX3D          | ON
528--   OptLevel    | 0
529-- Interviews    | ON
530--   PATH        | /Users/travis/build/neuronsimulator/nrn/external/iv
531--   INC         | /Users/travis/build/neuronsimulator/nrn/external/iv/src/include
532--   X11 (INC)   | /usr/X11R6/include
533-- CoreNEURON    | ON
534--   PATH        | /Users/travis/build/neuronsimulator/nrn/external/coreneuron
535--   LINK FLAGS  | -Lx86_64 -lcorenrnmech
```